wgpu/src/backend/web.rs

```diff
@@ -550,7 +550,6 @@ fn map_texture_format(texture_format: wgt::TextureFormat) -> web_sys::GpuTexture
         TextureFormat::Depth32FloatStencil8 => tf::Depth32floatStencil8,
         TextureFormat::Depth24Plus => tf::Depth24plus,
         TextureFormat::Depth24PlusStencil8 => tf::Depth24plusStencil8,
-        TextureFormat::Depth24UnormStencil8 => tf::Depth24unormStencil8,
```

Comment:
If this doesn't exist anymore, we shouldn't just remove it from the web backend - we should remove it everywhere.

Reply:
Is it worth if I do a pass on all TextureFormat then?

Reply:
Definitiely!

Reply:
Oki - working on updating TextureFormat (at least uncompressed ones)